services/app-api/libs/kafka-source-lib.js

```diff
@@ -40,7 +40,7 @@ class KafkaSourceLib {
   */
 
   unmarshallOptions = {
-    convertEmptyValues: true,
```

Comment:
this options does not exist in the new version; not sure if it fails us along the way

Reply:
I don't think it ever existed; this is a marshalling option, not an unmarshalling one. I don't believe it caused any negative effects... it would have just been ignored.